cmd/operator/main.go

```diff
@@ -101,20 +103,29 @@ func main() {
 		pm                    *patcher.Manager
 	)
 
-	ctrl.SetLogger(klogr.New())
+	ctrl.SetLogger(klogr.NewWithOptions(klogr.WithFormat(klogr.FormatKlog)))
 
 	flag.StringVar(&metricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&devicePluginNamespace, "deviceplugin-namespace", metav1.NamespaceSystem, "The namespace where deviceplugin daemonsets are created")
 	flag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
+	flag.BoolVar(&controllers.UpgradePlugins, "upgrade", true, fmt.Sprintf("Auto-upgrade plugins' images to the current release %s", controllers.ImageMinVersion))
+	flag.StringVar(&controllers.UpgradeToVersion, "upgrade-to-version", controllers.ImageMinVersion, "Upgrade / downgrade to official version, e.g. X.Y.Z, 0.23.0")
```

Comment:
Have I understood correctly that we're changing 2 variables in the controllers module this way?
Can we use local variables here and pass them along as parameters instead?